docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+# Most of this docker compose file was copied from here: 
+# https://github.com/graphprotocol/graph-node/blob/5ba079fff124f621b6beeba9dc39b81920a2f85b/docker/docker-compose.yml
+
+version: '3.9'
+services:
+  graph-node:
+    image: graphprotocol/graph-node
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
+      - postgres
+    extra_hosts:
+      - host.docker.internal:host-gateway
```

Comment:
This line is what allows the container to reach outside itself and into the host machine. For example if a non-dockerized service is running at localhost:8080, then the docker container can access it via `host.docker.internal:8080`

More info on that can be found [here](https://medium.com/@TimvanBaarsen/how-to-connect-to-the-docker-host-from-inside-a-docker-container-112b4c71bc66)

Reply:
Hmmm... For some reason, running macOS, it still looks like the node can't connect to the RPC. Using:

```
$ export ETH_MAINNET_ALCHEMY_RPC=http://127.0.0.1:8545
$ anvil --host 0.0.0.0
$ docker compose up
```

Anything I'm doing wrong? Can we get this in a README? 😄

Reply:
100%, i just added notes on using anvil in the README